### PR TITLE
DDF-04515 Fix Search Form Email Overflow Issue

### PR DESCRIPTION
**Pull request by willwill96**
#### What does this PR do?
Fixes an issue where e-mails were overflowing off of search/result forms.

Also, changes workspaces' e-mails overflow to use ellipses instead of ending abruptly at the end of the card

Adds a title property to the containing DOM elements so that the full e-mail can be viewed by hovering over it if needed.

See screenshots for more details.

#### Who is reviewing it? 
@gjvera 
@rymach 
@mojogitoverhere 
#### Ask 2 committers to review/merge the PR and tag them here.
@bdeining
@djblue
#### How should this be tested?
1. Quickbuild and unzip
2. Before starting up DDF, go to the etc/users.attributes file and change the e-mail to be longer
3. Start up DDF and verify nothing looks strange on the workspaces/search forms/result forms pages
#### What are the relevant tickets?
For Jira:
[](https://codice.atlassian.net/browse/)

For GH Issues:
Fixes: #4515 

#### Screenshots
Workspaces:
image

Search/Result Forms:
image

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Threat Dragon models
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: ui/packages/catalog-ui-search/src/main/webapp/react-component/presentation/workspace-item/workspace-item.tsx [hunk 1]
```diff
@@ -67,6 +67,12 @@ const Icon = styled.span`
   margin-right: calc(${props => props.theme.minimumSpacing} / 2);
 `
 
+const Author = styled.div`
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
```
Comment (bellcc) [inline ui/packages/catalog-ui-search/src/main/webapp/react-component/presentation/workspace-item/workspace-item.tsx]: ❓ Does this one need `display: block` too or can we remove that in the other `Author`?

**Comment by willwill96** [comment]
build now

**Comment by willwill96** [comment]
Unrelated Failure: build now

**Comment by willwill96** [comment]
build now

**Comment by willwill96** [comment]
build now

**Comment by willwill96** [comment]
build now

**Pull request merged**